trestle/oscal/__init__.py

```diff
@@ -15,5 +15,5 @@
 # limitations under the License.
 
 #TODO: Ensure this is automatically updated successfully.
-OSCAL_VERSION = '1.0.0'
-OSCAL_VERSION_REGEX = r'^1\.0\.0$'
+OSCAL_VERSION = '1.0.2'
+OSCAL_VERSION_REGEX = r'^1\.0\.[02]$'
```

Comment:
I get why it also matches OSCAL 1.0.0, however I am not sure this is correct. We don't fully support 1.0.0 anymore because of the AssessmentResults, so any model of that type would fail

Reply:
as discussed we need to continue to support 1.0.0 on import.